src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpExecutionServiceDAO.scala

```diff
@@ -17,29 +17,30 @@ import spray.httpx.SprayJsonSupport._
 class HttpExecutionServiceDAO( executionServiceURL: String )( implicit system: ActorSystem ) extends ExecutionServiceDAO {
 
   override def submitWorkflow( wdl: String, inputs: String, authCookie: HttpCookie ): ExecutionServiceStatus = {
-    val url = executionServiceURL+"/workflows"
+    // TODO: how to get the version?
```

Comment:
I imagine that's communicated verbally?  :smile:

 Probably all of your APIs are going to be using the same version, that should probably be a constant.